src/content/quickstart/openshift/ready_clusters.md

```diff
@@ -1,23 +1,28 @@
 ### Make your clusters ready for Submariner
 
-Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
-when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
-to the gateway nodes and ensuring that Pod IP addresses are preserved.
+Submariner gateway nodes need to be able to accept traffic over UDP ports (4500 and 500 by default) when using IPSEC.
+In addition we use port 4800/UDP to encapsulate traffic from the worker nodes to the gateway nodes and ensuring that Pod IP addresses are preserved.
 
 Additionally, the default OpenShift deployments don't allow assigning an elastic public IP
 to existing worker nodes, something that it's necessary at least on one end of the IPSEC connections.
 
-To handle all those details we provide a script that will prepare your AWS OpenShift deployment
-for Submariner, and will create an additional gateway node with an external IP.
+To handle all those details we provide a script
+[prep_for_subm.sh](https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh)
+that will prepare your AWS OpenShift deployment for Submariner, and will create an additional gateway node with an external IP.
+
+In the following example, we create the gateway node on cluster-b, with custom IPsec ports and instance type:
 
 ```bash
 
 curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
 chmod a+x ./prep_for_subm.sh
 
-./prep_for_subm.sh cluster-a     # respond yes when terraform asks
-./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+export IPSEC_NATT_PORT=4501
+export IPSEC_IKE_PORT=501
+export GW_INSTANCE_TYPE=m4.xlarge
+
```

Comment:
Can we create an additional section with those parameters, and one without?, those parameters are optional otherwise people will copy paste all of it (including the env vars).

See the example here:  https://github.com/submariner-io/submariner/tree/master/tools/openshift/ocp-ipi-aws

Reply:
done